src/app/record/components/funding/funding.component.html

```diff
@@ -135,8 +135,8 @@ <h3 class="orc-font-body-small" i18n="@@funding.amount">
         <span class="translated-title-label orc-font-body-small">
           <ng-container i18n="@@funding.translatedTitle"
             >Translated title</ng-container
-          >
-        </span>
+          > </span
+        >&nbsp;
```

Comment:
On the HTML we have avoided using this `&nbsp` before. Instead, we have added `preserWhiteSpaces` on the component configurations. 

Just competing this so you know is an alternative option.